modules/network-observability-config-flows-at-ingestion.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/configuring-operators.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-config-flows-at-ingestion_{context}"]
+
+= Filtering network flows at ingestion
+
+You can create filters to reduce the number of generated network flows. Filtering network flows can help reduce the resource usage of the Network Observability Operator.
+
+There are two different kinds of filters that you can configure:
+
+[id="ebpf-agent-filters_{context}"]
+== eBPF agent filters
+
+eBPF agent filters maximize performance because they take effect at the earliest stage of the Network Observability process.
+
+To configure eBPF agent filters with the Network Observability Operator, see "Filtering eBPF flow data using multiple rules".
+
+[id="flowlogs-pipeline-filters_{context}"]
+== Flowlogs-pipeline filters
+
+Flowlogs-pipeline filters provide greater control over traffic selection because they take effect later in the Network Observability process. They are primarily used to improve data storage.
+
+Flowlogs-pipeline filters use a simple query language to filter network flows:
+
+[source]
+----
+(srcnamespace="netobserv" OR (srcnamespace="ingress" AND dstnamespace="netobserv")) AND srckind!="service"
+----
+
+//05/15/2025: adding in syntax explanation for now. Checking with engineering on assumption of knowledge for target audience as I am not the target audience, so unclear if the target audience should already understand/have knowledge of/be able to read and understand syntax.
+////
+Syntax explanation:
+Logical boolean operators (case insensitive)
+and
+or
+Comparison operators
+equals =
+not equals !=
+matches regexp =~
+not matches regexp !~
+greater than (or equal) > / >=
+less than (or equal) < / <=
+Unary operations
+field is present: with(field)
+field is absent: without(field)
+Parenthesis-based priority
+////
+
+You can configure flowlogs-pipeline filters in the `spec.processor.filters` parameter of the `FlowCollector` resource. For example:
+
+.Example YAML Flowlogs-pipeline filter
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  agent:
+  processor:
+    filters:
+      - query: |
+          (SrcK8S_Namespace="netobserv" OR (SrcK8S_Namespace="openshift-ingress" AND DstK8S_Namespace="netobserv"))
+        outputTarget: Loki  <1>
+        sampling: 10  <2>
+----
+<1> Sends matching flows to a specific output, such as Loki, Prometheus, or an external system.
+<2> Optional. Applies a sampling ratio to limit the number of flows that match the query.
```

Comment:
Need more explanation about sampling. Will let @jotak put it into words

Reply:
maybe adding a short example like:

```suggestion
<2> Optional. Applies a sampling ratio to limit the number of matching flows to be stored or exported. For example, a value of 10 means 1 matching flow every 10 is kept.
```